app/navigation/AuthenticatedNavigator.scala

```diff
@@ -36,10 +36,11 @@ class AuthenticatedNavigator @Inject()() extends Navigator {
     case YourAddressPage => yourAddress(NormalMode)
```

Comment:
Should this be removed as in the same way we have dealt with this on PSubs, have the controller for Your address being hit to handle 423 locked and save the address then redirect to CYA?